src/views/SafeAreaView.js

```diff
@@ -88,17 +89,13 @@ class SafeView extends Component {
       return <View style={style}>{this.props.children}</View>;
     }
 
-    if (!forceInset && minor >= 50) {
```

Comment:
so we no longer use react-native's built-in SafeAreaView?

Reply:
Not for this version. It doesn't support padding.